automated install/basic-install.sh

```diff
@@ -1442,7 +1442,7 @@ installConfigs() {
             fi
             chmod 644 $conf
             if is_command lighty-enable-mod ; then
-                lighty-enable-mod pihole-admin access accesslog redirect fastcgi setenv > /dev/null || true
```

Comment:
This needs to go back here

Reply:
Yes. We need to use one of the options:

- or `lighty-enable-mod accesslog`;
- or  `server.modules = ( ..., "mod_accesslog")`.

Using both will generate "Duplicate config variable in conditional" error.
Using none will break the access log (and generate warnings about unknown options).